src/content/security/index.md

```diff
@@ -0,0 +1,231 @@
+
+#### Stay safe by:
+
+- Only install browser extensions from trusted sources
+- Removing unused browser extensions
+- Install Chrome extensions locally to stop auto-updating (Advanced)
+
+[More on the risks of browser extentions](<[https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/](https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/)>)
+
+<Divider />
+
+## Crypto Security 101
+
+### Upgrade Your Knowledge
+
+One of the biggest reasons people get scammed in crypto generally is a lack of understanding. If you don't understand that no one owns the Ethereum network, then it's easy to fall prey to the kind customer service agent who promises to return your lost ETH in exchange for your private keys. Educating yourself on how Ethereum works is a worthwhile investment.
+
+<DocLink to="/what-is-ethereum/" title="What is Ethereum?" />
+
+<DocLink to="/eth/" title="What is ether?" />
+
+## Wallet Security
+
+### Don't give out your Private keys
+
+**Never, for anyone reason, hand out your private keys!**
+
+The private key to your wallet acts as your digital signature. It is the only thing stopping someone who knows your wallet address from draining your account of all of its funds!
+
+### Use a Hardware Wallet
+
+A hardware wallet provides offline storage for private keys. They are considered the most secure wallet option for storing your private keys.
+
+Keeping private keys offline massively reduces the risk of being hacked, even if a hacker gets control of your computer.
+
+#### Try a hardware wallet:
+
+- [Ledger](<[https://www.ledger.com/](https://www.ledger.com/)>)
+- [Trezor](<[https://trezor.io/](https://trezor.io/)>)
+
+### Double check transactions
+
+Accidently sending crypto to the wrong wallet address is a common mistake. Unless you know the address owner, there will be no way for you to retrieve your funds.
+
+Always make sure the address you are sending to exactly matches the desired recipient's address before sending a transaction.
+
+### Set smart contract spend limits
+
+When interacting with smart contracts, do not allow unlimited spend limits. An unlimited spend could enable the smart contract to drain your wallet. Instead, set spending limits to only the amount necessary for the transaction.
+
+### Don't take screenshots of your seed phrases/private keys
+
+By screenshotting your seed phrases or private keys, you risk syncing them to the cloud and potentially making them accessible to hackers. Obtaining private keys from the cloud is a common attack vector for hackers.
+
+<Divider />
+
+## Common Scams
+
+Todo: add copy here
```

Comment:
I think we should mention something about rugpulls here too given the prominence of influencers promoting scam coins. Ex: Kim Kardashian and Floyd Mayweather promoting Ethereum Max.

Reply:
Good point. "Do your own research" is annoying but always useful advice. In this space you can't always rely on a "credible" person or organization to tell you what to do.